integration-tests/jest-reporter.js

```diff
@@ -32,7 +32,9 @@ module.exports = class JestReporter {
     const writableStream = createWriteStream(logFileName)
     writableStream.on('error', ({ message }) => console.error(`Error occurred while writing Console logs to ${logFileName}.\n${message}`))
 
-    _.forEach(({ message }) => writableStream.write(`${message}\n`), testResult.console)
```

Comment:
`_.forEach` is the problem. It didn't get every message in `testResult.console` array.